rtengine/procevents.h

```diff
@@ -1177,6 +1177,7 @@ enum ProcEventCode {
     Evlocallabsigjz = 1147,
     Evlocallabsigq = 1148,
     Evlocallablogcie = 1149,
+	EvLocallabavoidgamutMethod = 1150,
```

Comment:
It's better to create the procevent like you did here:
https://github.com/Beep6581/RawTherapee/blob/6f2ca3f2581e6f6dcecb00ffb6bb6a730988c972/rtgui/labcurve.cc#L39

That way, you can have a more descriptive history name like `HISTORY_MSG_AVOID_GAMUT_METHOD` instead of `HISTORY_MSG_1151`.

Reply:
Done